pint/unit.py

```diff
@@ -1109,6 +1110,9 @@ def _convert(self, value, src, dst, inplace=False):
         if isinstance(value, Decimal):
             factor = Decimal(str(factor))
 
+        if isinstance(value, Fraction):
+            factor = Fraction(str(factor))
```

Comment:
I'm not sure since it fails both with python2.6 and python2.7 (but differently):

```python
Python 2.6.9 (unknown, Jan 13 2016, 11:45:16) 
[GCC 4.9.2] on linux3
Type "help", "copyright", "credits" or "license" for more information.
>>> from fractions import Fraction
>>> Fraction(1.1)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/ekofman/.pyenv/versions/2.6.9/lib/python2.6/fractions.py", line 99, in __new__
    numerator = operator.index(numerator)
TypeError: 'float' object cannot be interpreted as an index
```

and:

```python
Python 2.7.9 (default, Mar  1 2015, 12:57:24) 
[GCC 4.9.2] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from fractions import Fraction
>>> Fraction(1.1)
Fraction(2476979795053773, 2251799813685248)
```
Which may also be the reason why the same code with Decimal also transforms the factor to a string and then to a Decimal ?